mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/index.tsx

```diff
@@ -38,10 +43,38 @@ function Analytics(): JSX.Element {
   );
 }
 
+function TopBarWithBreadcrumb(): JSX.Element {
+  const { automation } = useSelect((s) => ({
+    automation: s(editorStoreName).getAutomationData(),
+  }));
+
+  let status = __('Draft', 'mailpoet');
+  let statusClass = '';
+  if (automation.status === AutomationStatus.ACTIVE) {
+    status = __('Active', 'mailpoet');
+    statusClass = 'mailpoet-analytics-badge-success';
+  }
+  if (automation.status === AutomationStatus.DEACTIVATING) {
+    status = __('Deactivating', 'mailpoet');
+    statusClass = 'mailpoet-analytics-badge-warning';
+  }
+
+  return (
+    <TopBarWithBeamer>
+      <p className="mailpoet-automation-analytics-title">
+        {__('Automations', 'mailpoet')} › <strong>{automation.name}</strong>
```

Comment:
If it is a breadcrumb, I would expect the `{__('Automations', 'mailpoet')}` to be a link.
If it is a title, maybe use a heading tag for semantics?